services/aeroportos/index.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+import aeroportosList from './aeroportosList.json';
+
+export const getAeroportosData = async () => {
+  try {
+    const url =
+      'https://sistemas.anac.gov.br/dadosabertos/Aerodromos/Lista%20de%20aer%C3%B3dromos%20p%C3%BAblicos/AerodromosPublicos.json';
+    const { data: responseData } = await axios({
+      url,
+      method: 'get',
+    });
+    return responseData.map((objeto) => {
+      return {
+        CodigoOACI: objeto.CódigoOACI,
+        NomeAeroporto: objeto.Nome,
+        Cidade: objeto.Município,
+        Estado: objeto.UF,
+        LatGeoPoint: objeto.LatGeoPoint,
+        LonGeoPoint: objeto.LonGeoPoint,
```

Comment:
Aqui sugiro alterar como as chaves são acessadas e padronizar a saída para ficar no mesmo estilo dos demais endpoints
```suggestion
        codigo_oaci: objeto['CódigoOACI'],
        aeroporto: objeto['Nome'],
        cidade: objeto['Município'],
        estado: objeto['UF'],
        localizacao: {
          "type": "Point",
          "coordinates": {
            "longitude": parseFloat(objeto.['LonGeoPoint']),
            "latitude": parseFloat(objeto.['LatGeoPoint']),
          }
        }
```

Caso tenha interesse tem um PR que costumo usar como referência #268